### DISCUSSION TITLE
Deleting one block delets all blocks

### DISCUSSION THREAD

**Discussion by ex0tiqczs** [maruohon/litematica#890] [General]
Hey everyone, i have a huge problem with my Litematic.. if im going to delete one (missplaced) block, its going to delete their neighbourblocks as well.. anyone knows why? [Video on Medal here](https://medal.tv/de/games/minecraft/clips/iFdKJgvDnHAf-ZP0G/d1337MO7GBCw?invite=cr-MSxPWE4sNjA4Mzc0MTEs)

**Comment by maruohon** [comment]
Well that's not a Litematica feature at least. I was going to ask if the blocks are actually getting placed at all or if they are just client side ghost blocks, but the observers getting triggered must be a server side thing. Does this happen if you don't use Easy Place?
